CefSharp/AddressChangedEventArgs.cs

```diff
@@ -14,6 +14,11 @@ public class AddressChangedEventArgs : EventArgs
         public IBrowser Browser { get; set; }
         public string Address { get; private set; }
 
+        /// <summary>
+        /// Called when a frame's address has changed. 
+        /// </summary>
+        /// <param name="browser"></param>
```

Comment:
Most of the interfaces say `the browser object`, it may not be the best description ever, it's consistent at least.